### PR TITLE
use options object to configure Schism.Engine

### PR DESCRIPTION
**Pull request by rickbutton**
this should address https://github.com/google/schism/issues/88

### CODE REVIEW

**Comment by rickbutton** [comment]
@googlebot I signed it!

File: rt/rt.mjs [hunk 1]
```diff
@@ -203,14 +203,14 @@ class Module {
 }
 
 export class Engine {
-    constructor(filesystem) {
+    constructor(options) {
```
Comment (wingo) [inline rt/rt.mjs]: Might as well use destructuring here:

```
constructor({filesystem}) { ... }
```

Can be fixed in a follow-up if you like.

Comment (rickbutton) [reply rt/rt.mjs]: no problem! I just addressed it here.

**Comment by wingo** [comment]
thanks!

**Pull request merged**